nextjs-13/src/workers/sendMail.ts

```diff
@@ -0,0 +1,52 @@
+import Handlebars from 'handlebars';
+import { createTransport } from 'nodemailer';
+
+import { queryNewsletters } from 'repositories/newsletter.repository';
+
+const sendMail = async (job) => {
+  const { senderId, senderName, ids, to } = job.data;
+
+  const newsletters = await queryNewsletters(senderId, ids);
+  const items = newsletters.map((newsletter) => {
+    return {
+      id: newsletter.id,
+      name: newsletter.name,
+    };
+  });
+
+  const source =
+    '<p>Nimble Newsletter just invited you to view these newsletters:</p>' +
+    '<ul>' +
+    '{{#each items}}' +
+    `<li><a href="${process.env.NEXTAUTH_URL}/newsletter/{{this.id}}">{{this.name}}</a></li>` +
+    '{{/each}}' +
+    '</ul>';
```

Comment:
Wouldn't be more maintainable to have it defined in a a file `src/mailers/newsletterInvite.hbs`?

Reply:
Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/121/commits/87d08da65c49cf48665f91c966a6929500f8c9a2)